dex/src/main/scala/com/wavesplatform/dex/api/http/routes/MatcherApiRoute.scala

```diff
@@ -208,7 +209,8 @@ class MatcherApiRoute(
 
   private def placeOrder(endpoint: Option[PathMatcher[Unit]], isMarket: Boolean): Route = {
     val orderType = if (isMarket) "Market" else "Limit"
-    val route = (pathEndOrSingleSlash & post & measureResponse(s"place${orderType}Order") & protect & entity(as[Order])) { order =>
+    val route = (pathEndOrSingleSlash & post & withMetricsAndTraces(s"place${orderType}Order") & protect & entity(as[Order])) { order =>
+      setSpanNameAndForceSamplingDecision(s"/place${orderType}Order")
```

Comment:
It seems not needed, because we call this method in `withMetricsAndTraces`?

Reply:
yep, my fail

Reply:
fixed https://github.com/wavesplatform/matcher/pull/539/commits/9a6203cbec6d3cfbfeef380dbf38f1b92b96009d